pkg/api/speak-stream/v1/interfaces/types.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 Deepgram SDK contributors. All Rights Reserved.
+// Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+// SPDX-License-Identifier: MIT
+
+package interfaces
+
+import (
+	interfaces "github.com/deepgram/deepgram-go-sdk/pkg/client/interfaces"
+)
+
+/***********************************/
+// Request/Input structs
+/***********************************/
+type SpeakOptions interfaces.SpeakOptions
+
+type SpeakStreamResponse struct {
+	ContentType string `json:"content_type,omitempty"`
+	RequestID   string `json:"request_id,omitempty"`
+	ModelUUID   string `json:"model_uuid,omitempty"`
+	ModelName   string `json:"model_name,omitempty"`
+	Date        string `json:"date,omitempty"`
+}
+
+/***********************************/
+// MessageType is the header to bootstrap you way unmarshalling other messages
+/***********************************/
+/*
+	Example:
+	{
+		"type": "message",
+		"message": {
+			...
+		}
+	}
+*/
+type MessageType struct {
+	Type string `json:"type"`
+}
+
+// MetadataResponse is the response from the text-to-speech request
+type MetadataResponse struct {
+	Type      string `json:"type,omitempty"`
+	RequestID string `json:"request_id,omitempty"`
+}
+
+// FlushResponse is the response from the text-to-speech request which indicates that the server has flushed the buffer and is ready to return audio
+type FlushedResponse struct {
```

Comment:
This should be `FlushResponse` to match the label here:
https://github.com/deepgram/deepgram-go-sdk/pull/239/files#diff-e36b1ac7d75bbe05c7622c4e58a6a2c284fc55758034a4a28725cd2c8dbcea14R12